geomstats/geometry/discretized_curves.py

```diff
@@ -207,23 +229,22 @@ def exp(self, tangent_vec, base_curve):
 
         tangent_vec_derivative = (n_sampling_points - 1) * (
             tangent_vec[:, 1:, :] - tangent_vec[:, :-1, :])
-        base_curve_velocity = (n_sampling_points - 1) * (base_curve[:, 1:, :] -
-                                                         base_curve[:, :-1, :])
-        base_curve_velocity_norm = self.pointwise_norm(base_curve_velocity,
-                                                       base_curve[:, :-1, :])
+        base_curve_velocity = (n_sampling_points - 1) * (
+            base_curve[:, 1:, :] - base_curve[:, :-1, :])
+        base_curve_velocity_norm = self.pointwise_norm(
+            base_curve_velocity, base_curve[:, :-1, :])
 
-        inner_prod = self.pointwise_inner_product(tangent_vec_derivative,
-                                                  base_curve_velocity,
-                                                  base_curve[:, :-1, :])
+        inner_prod = self.pointwise_inner_product(
+            tangent_vec_derivative, base_curve_velocity, base_curve[:, :-1, :])
         coef_1 = 1 / gs.sqrt(base_curve_velocity_norm)
         coef_2 = -1 / (2 * base_curve_velocity_norm**(5 / 2)) * inner_prod
 
         term_1 = gs.einsum('ij,ijk->ijk', coef_1, tangent_vec_derivative)
         term_2 = gs.einsum('ij,ijk->ijk', coef_2, base_curve_velocity)
         srv_initial_derivative = term_1 + term_2
 
-        end_curve_srv = self.l2_metric.exp(tangent_vec=srv_initial_derivative,
-                                           base_landmarks=base_curve_srv)
+        end_curve_srv = self.l2_metric(n_sampling_points - 1).exp(
```

Comment:
Why `n_sampling_points - 1`?

Reply:
it was already the case
https://github.com/geomstats/geomstats/blob/932fe5a05da9d0f1622b51c152d3ab34d3e6ae3a/geomstats/geometry/discretized_curves.py#L230